src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void setupRegistrationService() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registration_validUser_ok() {
```

Comment:
```suggestion
    void register_validUser_ok() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.